doc/release/RELEASE-NOTES.md

```diff
@@ -3,6 +3,37 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). The format is based on the `RELEASE-NOTES-TEMPLATE.md` file.
 
+## Release 2.6.0
+
+## Introduction
+* Product name: Open Supply Hub
+* Release date: Jun 14, 2025
+
+### Database changes
+* *Describe high-level database changes.*
+
+#### Migrations:
+* *Describe migrations here.*
+
+#### Schema changes
+* *Describe schema changes here.*
+
+### Code/API changes
+* *Describe code/API changes here.*
+
+### Architecture/Environment changes
+* [OSDEV-1925](https://opensupplyhub.atlassian.net/browse/OSDEV-1925) - Deploy to Pre-prod on `release/*` branch creation is prohibited.
```

Comment:
- I recommend using the description you prepared for the PR instead of this one. That description is more informative, but please replace “The PR” with something more appropriate when transferring the description here.
- Use “releases” instead of “release”, and please fix the same issue in the PR description.